samples/boards/google_twinkie_v2_pda/src/view.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2023 The ChromiumOS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/smf.h>
+#include <zephyr/drivers/gpio.h>
+
+#include <zephyr/device.h>
+
+#include "meas.h"
+
+/* The devicetree node identifier for the led aliases. */
+#define LED0_NODE DT_ALIAS(led0)
+#define LED1_NODE DT_ALIAS(led1)
+#define LED2_NODE DT_ALIAS(led2)
+
+static const struct gpio_dt_spec led0 = GPIO_DT_SPEC_GET(LED0_NODE, gpios);
+static const struct gpio_dt_spec led1 = GPIO_DT_SPEC_GET(LED1_NODE, gpios);
+static const struct gpio_dt_spec led2 = GPIO_DT_SPEC_GET(LED2_NODE, gpios);
```

Comment:
How about using the LED APIs instead? Add `CONFIG_LED=y`, then you'd only have to get a device for the `gpio-leds` node and you could use the APIs from `zephyr/drivers/led.h` and drop the custom initialization code.